Library/Homebrew/missing_formula.rb

```diff
@@ -122,6 +122,8 @@ def tap_migration_reason(name)
           new_tap_user, new_tap_repo, new_tap_new_name = new_tap.split("/")
           new_tap_name = "#{new_tap_user}/#{new_tap_repo}"
 
+          break if OS.linux? && new_tap_name.start_with?("homebrew/cask")
```

Comment:
I think this should be fixed in Homebrew/linuxbrew-core instead where they should remove the cask migrations if they aren't desired CC @iMichka

Reply:
Ah, I see, [in tap_migrations.json](https://github.com/Homebrew/linuxbrew-core/blob/master/tap_migrations.json). I'm happy to do that if @iMichka agrees.

Reply:
I've deleted this line and have made the suggested change over in https://github.com/Homebrew/linuxbrew-core/pull/13277. :crossed_fingers: Thanks, Mike!